app/src/main/java/com/dessalines/thumbkey/ui/components/settings/lookandfeel/LookAndFeelActivity.kt

```diff
@@ -851,8 +816,44 @@ fun LookAndFeelActivity(
                         .value
                         .toInt().toString(),
                 )
+                SettingsCheckbox(
+                    state = slideEnabledState,
+                    icon = {
+                        Icon(
+                            imageVector = Icons.Outlined.SpaceBar,
+                            contentDescription = stringResource(R.string.slide_enable),
+                        )
+                    },
+                    title = {
+                        Text(stringResource(R.string.slide_enable))
+                    },
+                    onCheckedChange = {
+                        updateAppSettings(
+                            appSettingsViewModel,
+                            keySizeState,
+                            pushupSizeState,
+                            animationSpeedState,
+                            animationHelperSpeedState,
+                            minSwipeLengthState,
+                            slideSensitivityState,
+                            slideEnabledState,
+                            positionState,
+                            autoCapitalizeState,
+                            spacebarMultiTapsState,
+                            keyBordersState,
+                            vibrateOnTapState,
+                            soundOnTapState,
+                            hideLettersState,
+                            hideSymbolsState,
+                            keyboardLayoutsState,
+                            themeState,
+                            themeColorState,
+                        )
+                    },
+                )
                 SettingsSlider(
                     valueRange = 1f..50f,
+                    enabled = slideEnabledState.value,
```

Comment:
Nice, didn't know about this.

Reply:
Neither did I nor a good hours worth of googling. ChatGPT knew of it though haha.

Reply:
Neither did I nor a good hours worth of googling. ChatGPT knew of it though haha.